src/platforms/react-native/troubleshooting.mdx

```diff
@@ -242,3 +242,15 @@ React Native 0.60.3 tooling was unexpectedly moving source maps files, which cau
 ## Missing Java stacktrace on the New Architecture
 
 If you're using the new architecture on React Native, you might be missing the Java stacktrace. Follow the [issue](https://github.com/facebook/react-native/issues/34923) for more information.
+
+## Failing Build Due to `sentry-cli`
+
+If you're building your app without access to `sentry.io` or some other unexpected error occurs, you can allow failure of the `sentry-cli` step by adding an environmental variable `SENTRY_ALLOW_FAILURE` equal to `true`. The CLI will still run, but won't cause the build to fail:
```

Comment:
This may just be me, but "some other unexpected error" doesn't seem like a specific enough use case. An unexpected error occurring seems to be something that can happen in many situations. Can you add more context there?

Reply:
I just want to point out that the environment variable allows any failure of the cli not only connections to Sentry. It could be some issues with processing debug files. But maybe that’s obvious and I can leave that part out?

Reply:
Ah okay. I think it's just the word CLI that's missing there. Maybe something like:

> If you're building your app without access to `sentry.io` or some other unexpected CLI error occurs, ...